app/services/ersTemplatesInfo/EMITemplateInfo.scala

```diff
@@ -60,7 +60,7 @@ trait EMITemplateInfo {
     "3.Employee first name"	,
     "4.Employee second name(if applicable)",
     "5.Employee last name"	,
-    "6.National Insurance number(if applicable)"	,
+    "6.National Insurance number"	,
```

Comment:
in this file there are tabs/spaces after some " 
for example in this line
could be cleaned if there is not too much effort

Reply:
https://github.com/hmrc/ers-file-validator/pull/48/commits/d1ae8a63c65f3cf0dd49f40f09be471cf7dd28a7

Added changes